openedx/adg/lms/applications/signals.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.models import Group
```

Comment:
Let's rename the module to `handlers.py`, we use `signals.py` for custom signals